backend/entity/src/election_vote.rs

```diff
@@ -0,0 +1,33 @@
+//! `SeaORM` Entity. Generated by sea-orm-codegen 0.11.3
+
+use sea_orm::entity::prelude::*;
+
+#[derive(Clone, Debug, PartialEq, DeriveEntityModel, Eq)]
+#[sea_orm(table_name = "election_vote")]
+pub struct Model {
+    #[sea_orm(primary_key, auto_increment = false)]
+    pub election: i32,
+    #[sea_orm(primary_key, auto_increment = false)]
+    pub nomination_username: String,
+    pub count: i32,
+}
+
+#[derive(Copy, Clone, Debug, EnumIter, DeriveRelation)]
+pub enum Relation {
+    #[sea_orm(
+        belongs_to = "super::nomination::Entity",
+        from = "Column::Election",
+        to = "super::nomination::Column::Election",
+        on_update = "Cascade",
+        on_delete = "Restrict"
+    )]
+    Nomination,
```

Comment:
This needs to be tested; I'm not sure if the both columns are being used for the relation.

Reply:
Related: https://github.com/SeaQL/sea-orm/issues/1690